History.md

```diff
@@ -5,6 +5,9 @@
 * `meteor-tool@2.4`
   - `meteor show` now reports if a package is deprecated
 
+* `mongo@1.13.0`
+  - Add `createIndex` as a collection function. This is a new name for `_ensureIndex` which MongoDB has deprecated and removed in MongoDB 5.0. Use of `_ensureIndex` will show a deprecation warning on development.
```

Comment:
I believe it's important to mention that createIndex was added in 3. Otherwise people can think it's only going to work with Mongo 5.

Reply:
Will add. Right now I'm looking through our docs and I don't see `_ensureIndex` documented there, so I will add `createIndex` there as well.